SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -13,33 +13,52 @@ In order to interact with each other, Unreal and non-Unreal server-worker instan
+
+#### Using the code generation
+
+The external schema code generation tool generates C++ code in your game module from a specific directory containing schema files. The generated code includes classes representing each external schema enumeration and type, and an `ExternalSchemaInterface` class for sending updates and registering callbacks for receiving network operations. The `ExternalSchemaInterface` generates methods for each component and command within a component in your external schema.
+
+* For each component, the `ExternalSchemaInterface` will contain methods for `SendComponentUpdate`, `OnAddComponent`, `OnComponentUpdate`, `OnAuthorityChange`, and `OnRemoveComponent`.
+* For each command, the `ExternalSchemaInterface` will contain methods for `SendCommandRequest`, `SendCommandResponse`, `OnCommandRequest` and `OnCommandResponse`.
+
+##### Setting up
 
-#### Send data
-You set up your game to send SpatialOS component updates, command requests, and command responses directly using the `SpatialWorkerConnection.h` public methods:
+To interact with the external schema types by using the code generation tool, you'll need to do the following:
+
+* Run the `ExternalSchemaCodegen.bat` file as described in the [helper scripts]({{urlRoot}}/content/helper-scripts) documentation page to generate code from a directory containing external schema files.
+* Instantiate the `ExternalSchemaInterface` class inside your game module. If you want to ensure that the SpatialOS worker connection registers your callbacks to receive initial network operations, you need to register the callbacks inside your game instance's `OnConnected` event callback.
+* To send SpatialOS component updates and commands, call the overloaded `SendComponentUpdate`, `SendCommandRequest`, or `SendCommandResponse` method for the relevant schema type.
+* To register callbacks for receiving SpatialOS component network operations, call the overloaded `OnAddComponent`, `OnComponentUpdate`, `OnAuthorityChange`, `OnRemoveComponent`, `OnCommandRequest` or `OnCommandResponse` method for the relevant schema type.
+
+##### Generated Code Format
+
+// describe type implementation here
+
+#### Manual (de)serialization
+
+You can also manually serialize and deserialize C API network operations. This is what generated code alternative described above does under the hood. To do this, you'll need to do the following:
+
+To send SpatialOS component updates and commands, use methods defined in the `SpatialWorkerConnection.h` file:
 
 * `void SendComponentUpdate(Worker_EntityId EntityId, const Worker_ComponentUpdate* ComponentUpdate);`
 * `Worker_RequestId SendCommandRequest(Worker_EntityId EntityId, const Worker_CommandRequest* Request, uint32_t CommandId);`
 * `void SendCommandResponse(Worker_RequestId RequestId, const Worker_CommandResponse* Response);`
 
-You access these methods via a reference to the net driver, as shown below:
+which can access via a reference to the net driver:
```

Comment:
"can be accessed" or the previous version which uses active voice and is preferred by tech writers afaik